common/git-hooks/pre-commit

```diff
@@ -6,4 +6,8 @@
 # Invoke the "rush prettier" custom command to reformat files whenever they
 # are committed. The command is defined in common/config/rush/command-line.json
 # and uses the "rush-prettier" autoinstaller.
-node common/scripts/install-run-rush.js prettier || exit $?
\ No newline at end of file
+node common/scripts/install-run-rush.js prettier || exit $?
+if [ "$SKIP_GITLEAKS" != 1 ] 
```

Comment:
Should we be checking for `SKIP=gitleaks` instead, per their readme?

Reply:
The readme here refers to the way it would be disabled if the user installed it using [precommit](https://pre-commit.com/). 
But since it opened a comment, I will change it for consistency. Thanks!